webrender/build.rs

```diff
@@ -500,24 +500,6 @@ fn add_attribute_descriptors(
             );
             *vertex_offset += offset;
         }
-        // All shader files contain aData0 and aData1 variables.
-        // We must remove them from the attribute descriptors of blur and clip shaders,
-        // because they are not used in these and we don't want them when calculating offsets.
-        "aData0" | "aData1" => {
-            if !(file_name.starts_with("cs_blur") || file_name.starts_with("cs_clip")) {
```

Comment:
The `file_name` param becomes unused if you remove this.
Add `_` before it, or remove it from the params. Up to you.